graylog2-web-interface/src/components/configurations/UrlWhiteListConfig.jsx

```diff
@@ -0,0 +1,127 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Table } from 'components/graylog';
+import { IfPermitted } from 'components/common';
+import BootstrapModalForm from 'components/bootstrap/BootstrapModalForm';
+import UrlWhiteListForm from 'components/configurations/UrlWhiteListForm';
+import type { Config } from 'stores/configurations/ConfigurationsStore';
+
+
+type State = {
+  config: Config,
+  valid: boolean
+};
+
+type Props = {
+  config: Config,
+  updateConfig: (config: Config) => Promise<void>,
+};
+class UrlWhiteListConfig extends React.Component<Props, State> {
+  configModal: ?BootstrapModalForm;
+
+  inputs = {};
+
+  constructor(props: Props) {
+    super(props);
+    const { config } = this.props;
+    this.state = {
+      config,
+      valid: false,
+    };
+  }
+
+  _summary = (): React.Element<'tr'>[] => {
+    const { config: { entries } } = this.props;
+    return entries.map((urlConfig, idx) => {
+      return (
+        <tr key={urlConfig.id}>
+          <td>{idx + 1}</td>
+          <td>{urlConfig.title}</td>
+          <td>{urlConfig.value}</td>
+          <td>{urlConfig.type}</td>
+        </tr>
+      );
+    });
+  }
+
+  _openModal = () => {
+    if (this.configModal) {
+      this.configModal.open();
+    }
+  }
+
+  _closeModal = () => {
+    if (this.configModal) {
+      this.configModal.close();
+    }
+  }
+
+  _saveConfig = () => {
+    const { config, valid } = this.state;
+    const { updateConfig } = this.props;
+    if (valid) {
+      updateConfig(config).then(() => {
+        this._closeModal();
+      });
+    }
+  }
+
+  _update = (config: Config, valid: boolean) => {
+    const updatedState = { config, valid };
+    this.setState(updatedState);
+  }
+
+
+  _resetConfig = () => {
+    const { config } = this.props;
+    const updatedState = { ...this.state, config };
+    this.setState(updatedState);
+  }
+
+
+  render() {
+    const { config: { entries, disabled } } = this.props;
+    return (
+      <div>
+        <h2>URL Whitelist Configuration  {disabled ? <small>(Disabled)</small> : <small>(Enabled)</small> }</h2>
+
```

Comment:
I Agree with that, i myself think that we need more infos.

Reply:
I would suggest this:
```
<h2>URL Whitelist Configuration  {disabled ? <small>(Disabled)</small> : <small>(Enabled)</small> }</h2>
<p>
When enabled, outgoing HTTP requests from Graylog servers, such as event notifications or HTTP-based data adapter requests, are validated against the whitelists configured here.
Because the HTTP requests are made from the Graylog servers, they might be able to reach more sensitive systems than an external user would have access to, including AWS EC2 metadata, which can contain keys and other secrets, Elasticsearch and others.
Whitelist administrative access is separate from data adapters and event notification configuration.
</p>
```

Reply:
added